numpy/ma/tests/test_mrecords.py

```diff
@@ -370,48 +363,42 @@ def setUp(self):
     def test_view_by_itself(self):
         (mrec, a, b, arr) = self.data
         test = mrec.view()
-        self.assertTrue(isinstance(test, MaskedRecords))
+        assert_(isinstance(test, MaskedRecords))
         assert_equal_records(test, mrec)
         assert_equal_records(test._mask, mrec._mask)
 
     def test_view_simple_dtype(self):
         (mrec, a, b, arr) = self.data
         ntype = (np.float, 2)
         test = mrec.view(ntype)
-        self.assertTrue(isinstance(test, ma.MaskedArray))
+        assert_(isinstance(test, ma.MaskedArray))
         assert_equal(test, np.array(list(zip(a, b)), dtype=np.float))
-        self.assertTrue(test[3, 1] is ma.masked)
+        assert_(test[3, 1] is ma.masked)
 
     def test_view_flexible_type(self):
         (mrec, a, b, arr) = self.data
         alttype = [('A', np.float), ('B', np.float)]
         test = mrec.view(alttype)
-        self.assertTrue(isinstance(test, MaskedRecords))
+        assert_(isinstance(test, MaskedRecords))
         assert_equal_records(test, arr.view(alttype))
-        self.assertTrue(test['B'][3] is masked)
+        assert_(test['B'][3] is masked)
         assert_equal(test.dtype, np.dtype(alttype))
-        self.assertTrue(test._fill_value is None)
+        assert_(test._fill_value is None)
 
 
 ##############################################################################
-class TestMRecordsImport(TestCase):
-    # Base test class for MaskedArrays.
-    def __init__(self, *args, **kwds):
-        TestCase.__init__(self, *args, **kwds)
-        self.setup()
-
-    def setup(self):
-        # Generic setup
-        _a = ma.array([1, 2, 3], mask=[0, 0, 1], dtype=int)
-        _b = ma.array([1.1, 2.2, 3.3], mask=[0, 0, 1], dtype=float)
-        _c = ma.array([b'one', b'two', b'three'],
-                      mask=[0, 0, 1], dtype='|S8')
-        ddtype = [('a', int), ('b', float), ('c', '|S8')]
-        mrec = fromarrays([_a, _b, _c], dtype=ddtype,
-                          fill_value=(b'99999', b'99999.',
-                                      b'N/A'))
-        nrec = recfromarrays((_a._data, _b._data, _c._data), dtype=ddtype)
-        self.data = (mrec, nrec, ddtype)
+class TestMRecordsImport(object):
+
+    _a = ma.array([1, 2, 3], mask=[0, 0, 1], dtype=int)
+    _b = ma.array([1.1, 2.2, 3.3], mask=[0, 0, 1], dtype=float)
+    _c = ma.array([b'one', b'two', b'three'],
+                  mask=[0, 0, 1], dtype='|S8')
+    ddtype = [('a', int), ('b', float), ('c', '|S8')]
+    mrec = fromarrays([_a, _b, _c], dtype=ddtype,
+                      fill_value=(b'99999', b'99999.',
+                                  b'N/A'))
+    nrec = recfromarrays((_a._data, _b._data, _c._data), dtype=ddtype)
+    data = (mrec, nrec, ddtype)
```

Comment:
Setup is normally run for each test in a class, i.e., the class is reinstantiated for each test separately, or at least that was what I took from the documentation. But here II didn't seem that that was needed because the values were not modified in the tests and only needed to be set once. I think there are other tests that could be simplified the same way, but didn't want to take the time for a careful audit. That could be done later ...